src/desktop/add_extensions.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Categories=Settings;System;Qt;X-SuSE-YaST;X-SuSE-YaST-Software;
+
+X-SuSE-YaST-Call=scc
+X-SuSE-YaST-Argument=select_extensions
+X-SuSE-YaST-RootOnly=yes
+X-SuSE-YaST-Group=Software
+
+Icon=yast-product-registration
```

Comment:
I'd use another icon here, this is primarily for adding new extensions, the registration dialog is displayed only when the system is not registered yet.

I'd prefer the same icon as for the add-on module, i.e. `Icon=yast-addon`.